airgun/views/hostgroup.py

```diff
@@ -78,8 +79,7 @@ class host_group(SatTab):
     @View.nested
     class ansible_roles(SatTab):
         TAB_NAME = 'Ansible Roles'
-        resources = MultiSelect(id='ms-hostgroup_ansible_role_ids')
-        pagination = PF4Pagination()
```

Comment:
why removing Pagination widget here? 
It has been used in tests, removing it will fail those https://github.com/SatelliteQE/robottelo/pull/14914

Reply:
Added again.